app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -309,12 +326,16 @@ public Cursor buildSelectSql(int level, String fromTable) {
                     c1 = db.rawQuery(query, null);
                     break;
                 case LVL_ALBUM:
-                    query += "WHERE artist = ? group by album order by album";
+                    query += "WHERE artist = ? group by album order by ";
+                    query += orderAlbumsByYear ? "year, " : "";
```

Comment:
If you add the field that should be sorted to `LibraryAlbumOrder` you can access it here. Might be reuseable.